packages/jest-preset-mc-app/setup-tests.js

```diff
@@ -10,18 +10,27 @@ global.window.app = {
 
 const shouldSilenceWarnings = (...messages) =>
   [
-    /Warning: componentWillReceiveProps has been renamed/,
-    /CellMeasurerCache should only measure a cell's width or height/,
+    /.*Warning: componentWillReceiveProps has been renamed.*/,
+    /.*CellMeasurerCache should only measure a cell's width or height.*/,
```

Comment:
@ahmehri I think this is why it was never silenced in the first place. Cause it starts with warning and contains more stuff after.

Reply:
Actually it was silenced and it worked in many cases, not sure why it didn't work for you though.